modules/nf-core/pigz/compress/meta.yml

```diff
@@ -0,0 +1,41 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "pigz_compress"
+description: Compresses files with pigz.
+keywords:
+  - compress
+  - gzip
+  - parallelized
+tools:
+  - "pigz":
+      description: "Parallel implementation of the gzip algorithm."
+      homepage: "https://zlib.net/pigz/"
+      documentation: "https://zlib.net/pigz/pigz.pdf"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+
+  - raw_file:
+      type: file
+      description: File to be compressed
+      pattern: "*.*"
+
+output:
```

Comment:
Same as in xz, I think a note of the meta output should be added.

Reply:
Fixed in https://github.com/nf-core/modules/pull/4560/commits/94bf4d66b88b8f9031bd0042451e3e7ef3ddaf86